insights/tests/combiners/test_rhsm_release.py

```diff
@@ -57,6 +65,13 @@ def test_with_three():
     assert rhsm_release.minor == 6
 
 
+def test_with_three_but_rhsm_is_empty():
+    rhsm_release = RhsmRelease(RHSM_EMPTY_PARSER, SM_EMPTY_PARSER, RHUI_PARSER)
+    assert rhsm_release.set == '8.6'
+    assert rhsm_release.major == 8
+    assert rhsm_release.minor == 6
+
```

Comment:
The coverage test shows:
```
insights/combiners/rhsm_release.py                    22      0      8      1    97%   49->exit
```
 Accordingly, what about adding one more test case: `test_with_all_three_empty`, eg.
`rhsm_release = RhsmRelease(RHSM_EMPTY_PARSER, SM_EMPTY_PARSER, RHUI_EMPTY_PARSER)`.